packages/funding-rate-proposer/src/proposer.js

```diff
@@ -125,6 +133,18 @@ class FundingRateProposer {
     });
   }
 
+  async applyFundingRates() {
+    this.logger.debug({
+      at: "PerpetualProposer#applyFundingRates",
```

Comment:
I feel like we should create a task to go back through all our old log messages and make sure this `#at` syntax is correctly applied. It really helps with debugging

Reply:
yeah, you mean using the `#` should be more enforced?

Reply:
yup, exactly.